data/views/workflowTasks.2.0.json

```diff
@@ -0,0 +1,22 @@
+{
+    <% if (hasOwnProperty("friendlyName")) { %>
+    "friendlyName": "<%=friendlyName%>",
+    <% }%>
+    <% if (hasOwnProperty("injectableName")) { %>
+    "injectableName": "<%=injectableName%>",
+    <% }%>
+    <% if (hasOwnProperty("implementsTask")) { %>
+    "implementsTask" : "<%=implementsTask%>",
+    <% }%>
```

Comment:
As the Task-Schema is enabled for some task definition, so there is a new key `schemaRef` need be handled here.